.github/workflows/comments.yml

```diff
@@ -26,32 +20,31 @@ jobs:
         uses: xom9ikk/dotenv@v1.0.2
         with:
           path: .github/workflows/
+      - name: Move into From Bitnami
+        uses: peter-evans/create-or-update-project-card@v2
+        if: ${{ github.actor == 'bitnami-bot' }}
+        with:
+          project-name: Support
+          column-name: From Bitnami
+          token: "${{ secrets.BITNAMI_BOT_TOKEN }}"
       - name: Move into Pending
         uses: peter-evans/create-or-update-project-card@v2
-        if: ${{ (!contains(github.event.issue.labels.*.name, 'bitnami')) && contains(fromJson(env.BITNAMI_TEAM), github.event.comment.user.login) }}
+        if: ${{ github.actor != 'bitnami-bot' && (!contains(github.event.issue.labels.*.name, 'bitnami')) && contains(fromJson(env.BITNAMI_TEAM), github.actor) }}
         with:
           project-name: Support
           column-name: Pending
-          token: "${{ secrets.GITHUB_TOKEN }}"
+          token: "${{ secrets.BITNAMI_BOT_TOKEN }}"
       - name: Move into In Progress
         uses: peter-evans/create-or-update-project-card@v2
-        if: ${{ contains(github.event.issue.labels.*.name, 'in-progress') && (!contains(fromJson(env.BITNAMI_TEAM), github.event.comment.user.login)) }}
+        if: ${{ github.actor != 'bitnami-bot' && contains(github.event.issue.labels.*.name, 'in-progress') && (!contains(fromJson(env.BITNAMI_TEAM), github.actor)) }}
```

Comment:
Although I think is fine, careful, it's not the same `github.actor` than `github.event.comment.user.login`. The first one is the user who triggered the initial workflow run. The other one, is the user who wrote the comment. In this case we are based on comments event, is the same but if this event is triggered by any other event, the `github.actor`can be different than the author of the comment.